sdk/examples/how_tos/account/create.rs

```diff
@@ -23,10 +23,9 @@ async fn main() -> Result<()> {
         .with_storage_path(&std::env::var("WALLET_DB_PATH").unwrap())
         .finish()
         .await?;
-    let account = wallet.get_account("Alice").await?;
 
-    // May want to ensure the account is synced before sending a transaction.
-    let balance = account.sync(None).await?;
+    // May want to ensure the wallet is synced before sending a transaction.
+    let balance = wallet.sync(None).await?;
     println!("Accounts BEFORE:\n{:#?}", balance.accounts());
```

Comment:
accounts

Reply:
This is a valid one :)

Reply:
in the print we mean account outputs, not wallet accounts